packages/manager/apps/dedicated/client/app/ip/components/list/block/ip-ip-block.constant.js

```diff
@@ -3,3 +3,5 @@ export const ADDITIONAL_IP = 'Additional IP';
 export default {
   ADDITIONAL_IP,
 };
+
+export const DELETE_TRACKING_PREFIX = 'dedicated::ip::delete-additional-ip';
```

Comment:
Can you add it in the default export as well? And also can you place the default export at the end of the file?